mmeval/metrics/utils/util_3d.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# adapted from https://github.com/ScanNet/ScanNet/blob/master/BenchmarkScripts/util_3d.py # noqa
```

Comment:
Copy it as it is and put it in `mmeval/metrics/_vendor`

Reply:
moved